packages/@aws-cdk/cfnspec/CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+# CloudFormation Resource Specification v18.5.0
+
+## New Resource Types
+
+
+## Attribute Changes
+
+
+## Property Changes
+
+* AWS::Amplify::Branch EnablePerformanceMode (__added__)
+* AWS::AppStream::Fleet IamRoleArn (__added__)
+* AWS::AppStream::Fleet StreamView (__added__)
+* AWS::AppStream::ImageBuilder IamRoleArn (__added__)
+* AWS::EKS::Cluster KubernetesNetworkConfig (__added__)
+* AWS::StepFunctions::Activity Name (__deleted__)
+* AWS::StepFunctions::Activity Arn (__added__)
```

Comment:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1715#discussion_r496397195
https://github.com/aws-cloudformation/cfn-python-lint/pull/1722

Reply:
thanks @PatMyron!! - I've applied the same patch to our commit